config/config.sample.yml

```diff
@@ -63,7 +63,7 @@ common: &common
     - feedbacks.chat_user_id
     - feedbacks.created_at
     - feedbacks.updated_at
-    - feedbacks.id # Used to be feedbacks.is_ignored. Keep this as a placeholder so we don't break filters
+    - feedbacks.is_ignored
```

Comment:
Are these changes intended? `is_ignored` doesn't exist on the Feedback model anymore. Ditto the `ignored_users` re-addition below.

Reply:
Huh, not sure how I missed these.